content/work_out_income.md

```diff
@@ -182,4 +182,4 @@ When you take a large cash sum from your pension pot to pay for debts you also h
 These organisations offer impartial debt advice:
 
 - [The Money Advice Service](https://www.moneyadviceservice.org.uk/en/categories/debt-and-borrowing)
-- [The Citizens Advice service](http://www.adviceguide.org.uk/wales/debt_w/debt_help_with_debt_e.htm)
+- [Citizens Advice](http://www.adviceguide.org.uk/wales/debt_w/debt_help_with_debt_e.htm)
```

Comment:
https://www.citizensadvice.org.uk/debt-and-money/help-with-debt/ looks like a more up to date URL too. Not sure why we were using a page from the Welsh organisation.